### ISSUE TITLE
Process.Cmdline() makes it hard to parse individual arguments

### ISSUE THREAD

**Issue by cgilling** [shirou/gopsutil#165]
Hi, first off, thanks for writing this library, I've been looking for something like this for a while and finally stumbled upon this one.

I was wondering if there is some way that the info from `Cmdline` could be exposed as a `[]string` rather than just a `string`, the problem I have is that it makes it hard to get the real list of command line parameters, for example if I called a binary with something like:

```bash
./test 2342 "this is a single arg" hi there
```

I don't think I would be able to find out that `this is a single arg` was really just one argument

Perhaps a new function could be added to expose the it in slice form rather than just string form?

**Comment by cgilling** [comment]
From what I can tell from the `linux`, `darwin` and `freebsd` implementations this would be pretty easy to implements, because all the args are segmented by `byte(0)`, but the windows implementations perhaps not as easy and it just seems to be a strings. Would that be a non-starter?

**Comment by shirou** [comment]
Since #167 has been merged, I close this issue. Thank you so much!
